tools/autograd/gen_python_functions.py

```diff
@@ -26,8 +26,8 @@
     '_indexCopy_', 'max_values', 'min_values', 'argmax', 'argmin',
     '_cumsum.*', '_cumprod.*', '_sum.*', '_prod.*',
     '_th_.*', '_thnn_.*',
-    'arange.*', 'range.*', '_gesv.*', '_getri.*', '_inverse.*',
-    '_potrs.*', '_cholesky.*', '_btrifact.*',
+    'arange.*', 'range.*', '_solve.*', '_getri.*', '_inverse.*',
+    '_cholesky.*', '_btrifact.*',
```

Comment:
What happened to `'_potrs.*',` ?

Reply:
I forgot to remove this while renaming `potrs` to `cholesky_solve`, so this should not cause any issues whatsoever.